synadm/cli/__init__.py

```diff
@@ -33,6 +33,15 @@
 from synadm import api
 
 
+output_format_help = """'human' gives a tabular or list view depending on the
+fetched data. This mode needs your terminal to be quite wide! 'json' returns
```

Comment:
Sounds better to not refer to `human` output mode with "This mode"

```suggestion
fetched data. The 'human' mode needs your terminal to be quite wide! 'json' returns
```

(suggestion may need reformatting)

Reply:
Will change that. Thanks.

Reply:
I kind of rewrote that whole thing. Please review it again! Thanks!